product_docs/docs/pgd/5/upgrades/bdr_pg_upgrade.mdx

```diff
@@ -62,15 +62,38 @@ bdr_pg_upgrade [OPTION] ...
 In addition to the options for `pg_upgrade`, you can pass the following parameters
 to `bdr_pg_upgrade`:
 
+**Required parameters**
```

Comment:
Feels weird to both have required and optional section and also mark individual options as required

Reply:
Already pared down the entry with (REQUIRED) and (required) at the start and end of line. 

Will remove.